deepmd/infer/deep_pot.py

```diff
@@ -190,15 +191,33 @@ def __init__(
             )
 
     def _run_default_sess(self):
-        [self.ntypes, self.rcut, self.dfparam, self.daparam, self.tmap] = run_sess(
+        [
+            self.ntypes,
+            self.ntypes_spin,
+            self.rcut,
+            self.dfparam,
+            self.daparam,
+            self.tmap,
+        ] = run_sess(
             self.sess,
-            [self.t_ntypes, self.t_rcut, self.t_dfparam, self.t_daparam, self.t_tmap],
+            [
+                self.t_ntypes,
+                self.t_ntypes_spin,
```

Comment:
This will break other graphs which do not have this tensor.

Reply:
ok, we have resolved this problem.